qiskit/test/mock/fake_backend.py

```diff
@@ -18,13 +18,37 @@
 
 import uuid
 import warnings
+import json
+import os
+
+from typing import Iterable, List, Union
 
 from qiskit import circuit
-from qiskit.providers.models import BackendProperties
-from qiskit.providers import BackendV1, BaseBackend
+from qiskit.opflow.state_fns import dict_state_fn
```

Comment:
What is this needed for?

Reply:
I can't remember why this was added. But it's now removed.